### PR TITLE
Trev8939/BlendRenderer

### PR DESCRIPTION
**Pull request by TADraeseke**
Adds a sample demonstrating the use of BlendRenderer.

### CODE REVIEW

**Comment by TADraeseke** [comment]
As mentioned in [main issue ](https://devtopia.esri.com/runtime/java-android-sdk/issues/1312), this sample doesn't work beyond 100.1.0-1676. Throws the following exception on any attempt to change `mRasterLayer` to a `new RasterLayer`.

`com.esri.arcgisruntime.ArcGISRuntimeException: Object already owned.`

File: blend-renderer/src/main/java/com/esri/arcgisruntime/sample/blendrenderer/MainActivity.java [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+
+import java.io.File;
+import java.util.Arrays;
+
+public class MainActivity extends AppCompatActivity implements ParametersDialogFragment.ParametersListener {
+
+  private MapView mMapView;
+  private Raster mImageryRaster;
+  private Raster mElevationRaster;
+  private RasterLayer mRasterLayer;
```
Comment (doneill) [inline blend-renderer/src/main/java/com/esri/arcgisruntime/sample/blendrenderer/MainActivity.java]: Can be converted to local variable.

File: blend-renderer/src/main/java/com/esri/arcgisruntime/sample/blendrenderer/MainActivity.java [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+  private void updateRenderer() {
+    ColorRamp colorRamp = mColorRampType != ColorRamp.PresetType.NONE ? new ColorRamp(mColorRampType, 800) : null;
+    // if color ramp is not NONE, color the hillshade elevation raster instead of using satellite imagery raster color
+    mRasterLayer = colorRamp != null ? new RasterLayer(mElevationRaster) : new RasterLayer(mImageryRaster);
+    mMapView.getMap().setBasemap(new Basemap(mRasterLayer));
+      // create blend renderer
+      BlendRenderer blendRenderer = new BlendRenderer(
+          mElevationRaster,
+          Arrays.asList(9.0),
+          Arrays.asList(255.0),
```
Comment (doneill) [inline blend-renderer/src/main/java/com/esri/arcgisruntime/sample/blendrenderer/MainActivity.java]: Prefer `Collections.singletonList(255.0)`, immutable over `Arrays.asList()` to create a single item list.

**Comment by doneill** [comment]
I updated screenshot and links to data in README.

**Comment by TADraeseke** [comment]
@doneill I can't seem to login to access the shasta-elevation.zip

`The username you entered is not a member of this organization.`

**Comment by TADraeseke** [comment]
@doneill - Changes made, back to you!

**Comment by doneill** [comment]
@TADraeseke Updates look good, data should be accessible now.

**Comment by TADraeseke** [comment]
Add sample which demonstrates use of BlendRenderer.

**Pull request merged**